### PR TITLE
Fixes #24720: Update doc content for 8.1 release

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/24720

### CODE REVIEW

File: src/reference/modules/usage/pages/configuration_management.adoc [hunk 1]
```diff
@@ -227,6 +227,25 @@ Based on these facts, the compliance of a rule is calculated like this:
...
+
+Nodes have an individual score, represented by a letter from F (worst) to A (best).
+This score aims at aggregating information from various sources to give an overall
+view of its state.
+
+The node score is computed from several sub-scores:
+
+* The compliance score. It is computed based on the policies' compliance score, associating the score with a compliance percentage.
+* The system updates score. It is based on the number of available upgrades, with a special focus on security upgrades. It represents the score of the patch management for this node.
+* The vulnerability score. Complementary to the available updates scores, it shows the impact of current known vulnerabilities in installed packages. The score is computed using the number of vulnerabilities and their score.
```
Comment (ElaadF) [inline src/reference/modules/usage/pages/configuration_management.adoc:240]: Maybe we should specify that CVE plugin is required for this score

**Comment by ElaadF** [comment]
Vulnerabilities score is missing from your screenshot

File: src/reference/modules/usage/pages/configuration_management.adoc (hunk 1 above)
Comment (amousset) [reply src/reference/modules/usage/pages/configuration_management.adoc:240]: yes

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Review by ElaadF — APPROVED** [review]

**Pull request merged**